src/components/common/Button/Button.tsx

```diff
@@ -1,7 +1,6 @@
 import './_button.scss';
 
 type Props = {
-
```

Comment:
tutaj nie wywala błędu ?

Reply:
To jest partial w scss, więc działa poprawnie https://stackoverflow.com/questions/34889962/why-put-in-front-of-the-file-name-or-in-scss-css

Reply:
ok